app/grandchallenge/components/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+
+from grandchallenge.api.swagger import swagger_schema_fields_for_charfield
+from grandchallenge.cases.serializers import ImageSerializer
+from grandchallenge.components.models import (
+    ComponentInterface,
+    ComponentInterfaceValue,
+)
+
+
+class ComponentInterfaceSerialzer(serializers.ModelSerializer):
+    kind = serializers.CharField(source="get_kind_display", read_only=True)
+
+    class Meta:
+        model = ComponentInterface
+        fields = [
+            "title",
+            "description",
+            "slug",
+            "kind",
+            "pk",
```

Comment:
default_value not needed?

Reply:
Not yet! Will add it when we start to use them.